### PR TITLE
Server mongo db

### PR DESCRIPTION
**Pull request by lukepvb**
**Tasks Completed:** 

- MongoDB database setup, 
- config folder with keys.js file created to export mongoURI
- Required mongoURI in server.js, connected the database (db) 
- Created first iteration of our tripModel

Please review and send any notes you may have as far as data you would need on the front-end!

- Luke and Taylor

"... but at the end of the day"

### CODE REVIEW

**Review by infiniti33 — APPROVED** [review]
looks good!

File: server/models/tripModel.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      body: String,
+      date: Date
+    }
+  ],
+  stops: [
+    {
+      index: Number,
+      coordinates: [String],
+      next_stop: { type: Number, default: null },
+      previous_stop: Number,
```
Comment (armangurkan) [inline server/models/tripModel.js]: Please define the default value in this prop as well, the default value should be **null**

File: server/server.js [hunk 2]
```diff
@@ -1,81 +1,98 @@
...
+const path = require("path");
+const cloudinary = require("cloudinary");
+const formData = require("express-form-data");
 
 const PORT = 3000;
 
 const app = express();
 
-const userController = require('./controllers/controller');
+const userController = require("./controllers/controller");
```
Comment (armangurkan) [inline server/server.js:15]: please name the file userController if you are going to modify only User related transactions.

**Review by armangurkan — APPROVED** [review]

**Review by infiniti33 — APPROVED** [review]

File: server/server.js (hunk 2 above)
Comment (infiniti33) [reply server/server.js:15]: we haven't touched that file yet :)

**Review by infiniti33 — APPROVED** [review]

**Review by infiniti33 — APPROVED** [review]

**Review by infiniti33 — APPROVED** [review]

File: server/models/tripModel.js (hunk 1 above)
Comment (lukepvb) [reply server/models/tripModel.js]: Got it, thanks

**Review by infiniti33 — APPROVED** [review]

**Pull request merged**